packages/bot/src/stores/trade-animation-store.js

```diff
@@ -0,0 +1,37 @@
+import { observable, action, reaction } from 'mobx';
+
+export default class TradeAnimationStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+    }
+
+    @observable status_class = ['', '', ''];
+    @observable reactions = [];
```

Comment:
- Does this need to be `observable`?
- Better to rename this to `reaction_disposers`